lib/kubernetes-deploy/kubectl.rb

```diff
@@ -20,14 +20,15 @@ def initialize(namespace:, context:, logger:, log_failure_by_default:, default_t
       raise ArgumentError, "context is required" if context.blank?
     end
 
-    def run(*args, log_failure: nil, use_context: true, use_namespace: true, raise_if_not_found: false, attempts: 1,
-      output_is_sensitive: nil)
+    def run(*args, log_failure: nil, use_context: true, use_namespace: true, output: nil,
+      raise_if_not_found: false, attempts: 1, output_is_sensitive: nil)
```

Comment:
Exposing `output` as an actual parameter seems preferable to expecting callers to manually add the `--output` flag, which feels leaky to me.